ci/buildkite.py

```diff
@@ -151,9 +154,8 @@ def cmd_build(args, remainder):
 
     env = os.environ.copy()
     variant = plan['VARIANTS'][args.variant]
-    if 'env' in variant:
-        for key, value in variant['env'].items():
-            env[key] = str(value)
+    for key, value in variant.get('env', {}).items():
```

Comment:
I think it'd be better to let buildkite set the environment; it's a bit more clear what's going on.

Reply:
Not sure you're reading this right, this is saying, try to get the `env` dictionary, but if it's not there, just make an empty one. This is functionally the same as before (and in fact was a change you made, adding the `if 'env' in variant`, but I think this way is a bit more pythonic.

Reply:
I mean, I don't know that you need to be manipulating env here at all; I just left the code here to be careful.